packages/react-examples/.storybook/preview.js

```diff
@@ -163,20 +163,30 @@ function generateStoriesFromExamples(key, stories, req) {
 
   /** @type {string} */
   let componentName;
+
+  // https://storybook.js.org/docs/react/configure/sidebar-and-urls#permalinking-to-stories
+  // Story links should be the same per package and suite
+  /** @type {string} */
+  let componentId;
+
   if (segments.length === 3) {
     // ./ComponentName/ComponentName.Something.Example.tsx
     componentName = segments[1];
+    componentId = segments[1];
   } else {
     // ./package-name/ComponentName/ComponentName.Something.Example.tsx
     // For @fluentui/react, don't include the package name in the sidebar
     // @ts-ignore -- PACKAGE_NAME is replaced by a loader
     componentName = 'PACKAGE_NAME' === 'react' ? segments[2] : `${segments[2]} (${segments[1]})`;
+    componentId = segments[2];
   }
 
+  console.log(componentName, componentId);
```

Comment:
Should this be here?

Reply:
Should what be there ? 😛 

```suggestion
```

Reply:
Damm. that's what I call fast in the trigger 😂